fuzzing/private/binary.bzl

```diff
@@ -198,6 +206,10 @@ be incorporated in the target configuration (e.g., on the command line or the
             doc = "A dictionary file to use in fuzzing runs.",
             allow_single_file = True,
         ),
+        "options": attr.label(
+            doc = "A file with INI-style options for OSS-Fuzz.",
+            allow_single_file = True,
+        ),
```

Comment:
I don't think this duplication scales. Should we define the shared attributes in a separate dict variable that we merge into each rule definition?

Reply:
I did using skylib's `dicts.add`.